Sources/VitaminCore/Foundations/Assets/VitaminAssetsFlags.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import Foundation
+
+extension VitaminAssets {
+    /// Return the country flag using the country code.
+    /// - Parameter countryCode: alpha-2 country code (ISO 3166-1).
```

Comment:
Some of the flag assets have a three-character name (`CAF`, `CAS`, `CEU`, `CNA`, `COC`, `CSA`, `DOM`, `IND`, `ISL`)
Should we rename the assets (with a potential breaking change) ?
Should we have a correspondance table between actual codes and assets names ?
Should we just document these specific cases ?

Reply:
Well seen! I think that it's not normal to have 2 different kinds of country codes 😕 I'll use the alpha-2 country code everywhere but without breaking changes (just some deprecated notices).
There are also some no country flags, so for these, I think that we can just ignore them and use them like before. 
It's good for you?

Reply:
Ok for me, but VitaminAssets.swift is a SwiftGen generated class... So I think we should avoid tampering with it.
How will you handle the deprecation ? With an VitaminAssets extension where you will hardcode the old three-character country codes (DOM, IND, ISL) ?
If yes, that's seems me ok.

And ok for the other "exotic" sub country flags that will be accessible the old way, or by forcing a non ISO code in the new API

Reply:
I just update the PR, I made an extension outside of the SwiftGen class to make a link to the new image name.